src/components/ConfigEditor.tsx

```diff
@@ -69,6 +79,15 @@ export const BigQueryConfigEditor: React.FC<BigQueryConfigEditorProps> = (props)
             menuShouldPortal={true}
           />
         </Field>
+        <Field label="Custom API endpoint" description={<span>Use a custom API endpoint.</span>}>
```

Comment:
I think more meaningful description would help.
```suggestion
        <Field label="Service endpoint" description={<span>
              Specifies the network address of an API service. Read more about service endpoint{' '}
              <a
                href="https://cloud.google.com/bigquery/docs/reference/rest#service-endpoint"
                rel="noreferrer"
                className="external-link"
                target="_blank"
              >
                here
              </a>
            </span>}>
```